proposals/description/tip_121b.ts

```diff
@@ -0,0 +1,164 @@
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Move all DAI and WETH to daiHoldingPCVDeposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.daiHoldingPCVDeposit, // pcvDeposit
+        addresses.weth, // token
+        addresses.tribalCouncilSafe, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: `
+      Move all WETH from daiHoldingPCVDeposit to the Tribal Council safe Multisig,
+      in preparation for selling
+      `
+    },
+
+    // 3. Cleanup FEI/TRIBE on TC Timelock from the Rari Infra team clawback
+    {
+      target: 'tribe',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribalCouncilTimelock, addresses.core, TC_TRIBE_BALANCE],
+      description: 'Move all 2.7M TRIBE from TC Timelock to DAO Treasury'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribalCouncilTimelock, addresses.feiDAOTimelock, TC_FEI_BALANCE],
+      description: 'Move all 2.7M FEI from TC Timelock to DAO Timelock'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'burn(uint256)',
+      arguments: (addresses) => [TC_FEI_BALANCE],
+      description: 'Burn 2.7M FEI from the DAO Timelock'
+    },
+    // 4. Update CR oracle to use stETH oracle for stETH rather than ETH oracle
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.weth, addresses.chainlinkStEthUsdOracleWrapper],
+      description: 'Update the ETH oracle in the CR oracle, to the stETH oracle as remaining ETH will be stETH'
+    },
+    // 5. Update CR oracle
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'removeDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.aaveFeiPCVDepositWrapper,
+          addresses.compoundFeiPCVDepositWrapper,
+          addresses.wethHoldingPCVDeposit,
+          addresses.lusdToDaiLensLusd,
+          addresses.lusdToDaiLensDai,
+          addresses.ethToDaiLensDai,
+          addresses.ethToDaiLensEth,
+          addresses.tribalCouncilTimelockFeiLens
+        ]
+      ],
+      description: 'Remove deprecated/empty smart contracts from CR Oracle'
+    },
+    // 6. Remove non-needed safe addresses
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'unsetSafeAddresses(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.voltHoldingPCVDeposit,
+          addresses.gOHMHoldingPCVDeposit,
+          addresses.lusdHoldingPCVDeposit,
+          addresses.wethHoldingPCVDeposit
+        ]
```

Comment:
Can also unset `ethToDaiLBPSwapper`

Reply:
don't do this yet, keep lusd and weth for now too